NuRadioReco/modules/phasedarray/phasedArray.py

```diff
@@ -0,0 +1,215 @@
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.utilities import units
+import logging
+import scipy
+import numpy as np
+from scipy import constants
+
+logger = logging.getLogger('phasedTriggerBase')
```

Comment:
Please change logger name to inherit log level from top module, i.e., `NuRadioReco.modules.trigger....`